auto-update/Control/Debounce.hs

```diff
@@ -25,70 +26,34 @@
 -- @since 0.1.2
 module Control.Debounce
     ( -- * Type
-      DebounceSettings
+      DI.DebounceSettings
     , defaultDebounceSettings
       -- * Accessors
-    , debounceFreq
-    , debounceAction
+    , DI.debounceFreq
+    , DI.debounceAction
+    , DI.debounceEdge
+    , DI.DebounceEdge(..)
```

Comment:
Is it conceivable that we'd want to add more data constructors here in the future? If so, I'd rather export value-level identifiers for each constructor to avoid a breaking change if we add more options. However, I can't think of any reasonable additions myself.

Reply:
Well, I was taking inspiration from Lodash's API. Looking at Lodash's manual, this seems to have been how they do it for the last 7 years so my thinking was that this API is tried and true. But happy to change to the identifiers if you prefer.

Reply:
That seems fine to me then, we can leave as-is.